src/app/table/table.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, EventEmitter, Input, OnInit, Output, TemplateRef } from '@angular/core';
+import { ApiService } from '../api.service';
+import { TableconfigService } from './tableconfig.service';
+
+@Component({
+  selector: 'app-table',
+  templateUrl: './table.component.html',
+  styleUrls: ['./table.component.css'],
+})
+export class TableComponent {
+  @Input() colHeader: any;
+  @Input() tableData: any;
+  @Input() notEmptyPost: any;
+  @Input() notscrolly: any;
```

Comment:
I think 'notscrolly' is unnecessary input variable. You can stop scrolling using notMorePost variable